File: 3_frontend/part13/MAGUS PRIME X 2_server.js
const express = require("express");
const path = require("path");
const app = express();
const port = 3000;

// Serve static files from the current directory
app.use(express.static(path.join(__dirname)));

// Start the server
app.listen(port, () => {
  console.log(`MAGUS PRIME X test server running at http://localhost:${port}`);
  console.log(
    `Open your browser to http://localhost:${port}/test-navbar-buttons.html`,
  );
});

File: 1_py/part7/MAGUS PRIME X 2_server.py
import http.server
import socketserver

PORT = 8085


class MyHTTPRequestHandler(http.server.SimpleHTTPRequestHandler):
    def do_GET(self):
        # If root path or index.html is requested, serve the startup_animation.html
        if self.path == "/" or self.path == "/index.html":
            self.path = "/startup_animation.html"
        return http.server.SimpleHTTPRequestHandler.do_GET(self)


Handler = MyHTTPRequestHandler

with socketserver.TCPServer(("", PORT), Handler) as httpd:
    print(f"Serving at http://localhost:{PORT}")
    print("Open your browser to see the MAGUS PRIME X animated intro")
    httpd.serve_forever()
